docs/troubleshooting.md

```diff
@@ -251,3 +251,7 @@ AWS Lambda [limits](https://docs.aws.amazon.com/lambda/latest/dg/limits.html) de
 - https://github.com/adieuadieu/serverless-chrome/blob/master/docs/chrome.md (tracks the latest Chromium snapshots)
 - https://github.com/sambaiz/puppeteer-lambda-starter-kit (uses an old version of Chromium)
 - https://github.com/universalbasket/aws-lambda-chrome
+
+## Running Puppeteer on browserless
```

Comment:
IMO, it feels more appropriate to list this as an example under https://github.com/GoogleChrome/puppeteer/tree/master/examples#other-resources instead of a way to "troubleshoot" issues in Puppeteer.

Instead of highlighting the service, we highlight the open source project and users can find there way to service via your github project.